Tests/MongoSwiftTests/MongoCollection+IndexTests.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+@testable import MongoSwift
```

Comment:
does this need to be a `@testable`? I don't think we are using any internal client API

Reply:
same as above